examples/clue_ams_remote.py

```diff
@@ -0,0 +1,73 @@
+"""
+This example solicits that apple devices that provide notifications connect to it, initiates
+pairing, then allows the user to use a CLUE board as a media remote through both the buttons
+and capacitive touch pads.
+"""
```

Comment:
I would prefer if you could include a note in the doc string about the separate libraries needed to run this example. Whenever I include an example in a library-specific repo that requires an atypical library (e.g. adafruit_ble, and adafruit_ble_apple_media), I include a note in the docstring to alleviate any potential confusion.

Reply:
Will do